src/dragonBones/armature/Slot.cpp

```diff
@@ -31,14 +31,7 @@ void Slot::_onClear()
 
     for (const auto& pair : disposeDisplayList)
     {
-        if (pair.second == DisplayType::Armature)
```

Comment:
This.... feels like a hack to me. After adding the nested armatures; this was causing Godot to crash when unloading everything. I think it was due to the order of which nodes are removed in Godot vs what the dragonbones API was expecting.

I checked for memory leaks and this seemed safe. Might be an issue if we start removing slots at runtime; though that isn't supported at the moment